ACMILabs/transcoder
# #23 Add exhibitions transcoder flags

Resolves #23

- [x] Add an environment variable for an `exhibitions` transcoder
- [x] Update ffmpeg settings to those below

Reference: https://superuser.com/a/1136305

## Testing

1. Create a `dev.env` file: `cp dev.tmpl.env dev.env`
1. Add the watch folders to your filesystem... e.g.

```
SMB_WATCH=/Users/sloffler/Code/transcoder-test-folders/watch
SMB_ACCESS=/Users/sloffler/Code/transcoder-test-folders/access
SMB_WEB=/Users/sloffler/Code/transcoder-test-folders/web
SMB_MASTER=/Users/sloffler/Code/transcoder-test-folders/master
SMB_OUTPUT=/Users/sloffler/Code/transcoder-test-folders/output
```

1. Build the project: `make build`
1. Run tests: `docker exec -it transcoder_transcoder_1 make test`
1. Stop the transcoder
1. Make a [short video file](https://acmiteam.slack.com/files/UBJ7W5H8F/F01C8RUU8BY/acmi.mp4) with the name: `B2004203_mo01_AmazingShortVideo.mp4`
1. Add it to your `watch` folder
1. Start XOS locally `docker-compose up`
1. Start the transcoder: `make up`
1. Check the video uploaded to your local XOS is the right aspect ratio, size, frames per second, and attempts to have a 20k bitrate: http://localhost:8000/admin/assets/video/?o=-5.3
